From e43c44017f733604d734d98348dc1c810f9e51b6 Mon Sep 17 00:00:00 2001
From: Victor Rodriguez <victor.rodriguez.bahena@intel.com>
Date: Wed, 29 Nov 2017 18:25:58 +0000
Subject: [PATCH] Fix missing depndencies for nervananeon package

missing dependencies for neon package 2.4 shown in machine-learning-basic bundle

not ok 13 machine-learning-basic pip3 check

 `pip3 check' failed
 nervananeon 2.4.0 requires pytest-mock, which is not installed.
 nervananeon 2.4.0 requires filelock, which is not installed.
 nervananeon 2.4.0 requires py-cpuinfo, which is not installed.
 nervananeon 2.4.0 requires pypandoc, which is not installed.
 nervananeon 2.4.0 requires pandoc, which is not installed.

Signed-off-by: Victor Rodriguez <victor.rodriguez.bahena@intel.com>
---
 requirements.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index b3f86f3..f153fdb 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -6,7 +6,6 @@ flake8==3.0.4
 funcsigs==1.0.2
 pytest==3.0.1
 pytest-cov==2.3.1
-pytest-mock==1.6.0
 posix_ipc==1.0.0
 pillow==3.3.1
 pylint==1.6.4
@@ -16,7 +15,3 @@ appdirs==1.4.3
 future==0.15.2
 tqdm==4.8.4
 cffi==1.11.0
-filelock==2.0.8
-py-cpuinfo==3.2.0
-pypandoc==1.4
-pandoc==1.0.2
-- 
2.15.0

